.vscode/launch.json

```diff
@@ -10,8 +10,8 @@
       "type": "extensionHost",
       "request": "launch",
       "runtimeExecutable": "${execPath}",
-      "args": ["--extensionDevelopmentPath=${workspaceRoot}"],
-      "outFiles": ["${workspaceRoot}/client/out/**/*.js"]
+      "args": ["--extensionDevelopmentPath=${workspaceFolder}/clients/vscode/"],
+      "outFiles": ["${workspaceFolder}/clients/vscode/out/**/*.js"]
```

Comment:
What would happen if we launch vscode client without running the server? Is it expected to work with the existing server code?

Reply:
Do you mean: Is it possible to run the `Launch client` config without running `Attach to Server` config? 
- that is possible, but in this case you can not debug the server (i.e. go to breakpoints in server code)